src/main/java/com/afornalik/ox/model/board/Board.java

```diff
@@ -78,8 +79,23 @@ public int receiveNumberOfEmptyFields() {
     private boolean checkIndexRange(int indexOfField) throws OutOfBoardException {
         int minValue = 0;
         if (indexOfField < minValue || indexOfField > maxValue.intValue()) {
-            throw new OutOfBoardException("Out of board - min value is : " + minValue + ", and max value is : " + maxValue.toString());
+            throw new OutOfBoardException("Out of board - min value is : " + (minValue+1) + ", and max value is : " + maxValue.add(BigInteger.ONE).toString());
```

Comment:
Are you sure you have to use BigInteger here? How big your board is supposed to be?

Reply:
At first i though about very large number, actually i changed type to int.